cilantro_audit/completed_audits_list_page.py

```diff
@@ -283,6 +283,9 @@ def build_completed_audit_page_body(self, completed_audit):
         self.manager.get_screen(COMPLETED_AUDIT_PAGE).stack_list.clear_widgets()
         self.manager.get_screen(COMPLETED_AUDIT_PAGE).stack_list.height = 0
         self.manager.get_screen(COMPLETED_AUDIT_PAGE).reset_scroll_to_top()
+        self.manager.get_screen(COMPLETED_AUDIT_PAGE).at = audit_template
```

Comment:
Unresolved reference to audit_template, crashes when opening a completed audit in admin view.

Reply:
Resolved